### PR TITLE
Conformance results for v1.28/rh-device-edge

### PR DESCRIPTION
**Pull request by eslutsky**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [ ] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form)?
* [X] Did you include the product/project logo in SVG, EPS or AI format?  
* [X] Does your logo clearly state the name of the product/project and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [X] If your product/project is open source, did you include the `repo_url`?
* [X] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

For a full list of requirements, please refer to these sections of the docs: [_Contents of the PR_](https://github.com/cncf/k8s-conformance/blob/master/instructions.md#contents-of-the-pr), and [_Requirements_](https://github.com/cncf/k8s-conformance/blob/master/instructions.md#requirements).

### CODE REVIEW

File: v1.28/rh-device-edge/README.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# Red Hat Device Edge
+
+## Create a multi-node MicroShift cluster
+
+### Install two MicroShift hosts
+
+1. Prepare two RHEL 9.2 hosts suitable for running MicroShift. The following configuration is recommended:
+- RHEL 9.2
```
Comment (DanielFroehlich) [inline v1.28/rh-device-edge/README.md:8]: or 9.3

**Review by DanielFroehlich — APPROVED** [review]

File: v1.28/rh-device-edge/README.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+5. Run the following commands to copy the configuration script to the secondary host and run it remotely.
+
+```
+cd ~/microshift/
+scp -o StrictHostKeyChecking=no ./scripts/multinode/configure-sec.sh redhat@${SEC_ADDR}:
+ssh redhat@${SEC_ADDR} ./configure-sec.sh "${PRI_HOST}" "${PRI_ADDR}" "${SEC_HOST}" "${SEC_ADDR}"
+```
+
+6. Set the KUBECONFIG variable using the configuration file from the primary host by runnign the following commands.
```
Comment (jerpeter1) [inline v1.28/rh-device-edge/README.md:90]: ... by running the following

**Comment by eslutsky** [comment]
the attached the `e2e.log` and `junit_01.xml` are taken from the [artifact](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/logs/periodic-ci-openshift-microshift-release-4.15-ocp-metal-nightly/1764854350863142912/artifacts/ocp-metal-nightly/openshift-microshift-e2e-metal-tests/artifacts/scenario-info/el92-src@cncf-conformance/) of automated CI [job](https://github.com/openshift/microshift/blob/release-4.15/test/scenarios-periodics/el92-src@cncf-conformance.sh#L41).

**Comment by eslutsky** [comment]
the attached result files missing the `[Serial]` tests,because we run our  CI jobs with `e2e-parallel` set.

**Review by Cmierly — APPROVED** [review]

**Comment by Cmierly** [comment]
You are now Certified Kubernetes

**Pull request merged**